### PR TITLE
Fix changelog

### PR DESCRIPTION
**Pull request by MetRonnie**

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ ones in. -->
 
 ## __cylc-rose-1.3.3 (<span actions:bind='release-date'>Released 2024-04-05</span>)__
 
-[#300](https://github.com/cylc/cylc-rose/pull/300) -
+[#302](https://github.com/cylc/cylc-rose/pull/302) -
 Fix issues which could cause "fcm_make" and "rose_prune" tasks intermittently
 fail with the message
 "Workflow database is incompatible with Cylc x.y.z, or is corrupted".
```
Comment (MetRonnie) [inline CHANGES.md:14]: Is the description correct?

Comment (oliver-sanders) [reply CHANGES.md:14]: Yes, this description is correct.

Comment (oliver-sanders) [reply CHANGES.md:14]: For reference, it is coming from here:

https://github.com/cylc/cylc-flow/blob/4b70574c9100e1cf830a226c254cb3159a558873/cylc/flow/workflow_db_mgr.py#L704

The `OperationalError` exception includes locking issues.

**Comment by MetRonnie** [comment]
@oliver-sanders Happy with this?

**Review by oliver-sanders — APPROVED** [review]

**Pull request merged**